std/zlib.d

```diff
@@ -75,9 +75,9 @@ enum
 
 class ZlibException : Exception
 {
-    this(int errnum)
-    {   string msg;
-
+    private static string getmsg(int errnum) nothrow @nogc pure
```

Comment:
Could be `@safe` too (though the entire module isn't @safe yet anyhow)

Reply:
I agree. Done.